### ISSUE TITLE
Tests fail on v0.7.1

### ISSUE THREAD

**Issue by mister-walter** [milouse/chwall#4]
Many thanks for developing chwall!

I use chwall on Arch Linux and ran into an issue when using `yay` to update it from v0.6.0-3.

The issue still occurs when I clone this repo directly and run `make test`, so it doesn't appear to be makepkg-specific.

Four of the tests fail. I've attached the relevant output below, but it appears that it's expecting different values in certain parts of a generated service file. Maybe I need to uninstall chwall for the tests to pass?

For convenience, here is the output for one of the failing tests:
```
FAIL: test_04_create_local_systemd_service_file_from_client (tests.test_local_desktop_files.TestLocalDesktopFiles.test_04_create_local_systemd_service_file_from_client)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.11/unittest/mock.py", line 1375, in patched
    return func(*newargs, **newkeywargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drew/tmp/chwall-upstream/tests/test_local_desktop_files.py", line 57, in test_04_create_local_systemd_service_file_from_client
    self.assertEqual(mock_stdout.getvalue(), result)
AssertionError: 'Serv[320 chars]very 3600 seconds\nAfter=network.target\n\n[Ti[79 chars]et\n' != 'Serv[320 chars]very 600 seconds\nAfter=network.target\n\n[Tim[76 chars]et\n'
  Service file:
  [Unit]
  Description=Simple wallpaper changer
  After=network.target
  
  [Service]
  Type=simple
  Environment=DISPLAY=:0
  ExecStart=python -m chwall.client next no_restart
  WorkingDirectory=/home/drew/tmp/chwall-upstream
  
  [Install]
  WantedBy=default.target
  
  Timer file:
  [Unit]
- Description=Change wallpaper every 3600 seconds
?                                    -
+ Description=Change wallpaper every 600 seconds
  After=network.target
  
  [Timer]
- OnActiveSec=3600
?             -
+ OnActiveSec=600
- OnUnitActiveSec=3600
?                 -
+ OnUnitActiveSec=600
  
  [Install]
  WantedBy=timers.target
```

[test-output.log](https://github.com/milouse/chwall/files/14620200/test-output.log)

Thanks again for developing `chwall`! Let me know if I can provide any more info.

**Comment by milouse** [comment]
Hi, and thank you very much for your feedback.

You are right, there is something wrong with the tests. The tests are wrongly taking some values from your own config file. I’ll try to fix that and release a new version to fix that.

In the mean time, you can still install the new version if you remove (or comment) the `check` function in the PKGBUILD before actually building the package (I don’t know yay, thus I let you check in their documentation how to do so).

**Comment by milouse** [comment]
Hi again, it should be fixed by my last release v0.7.2, already published on aur. Can you confirm if you have time? Thank you very much.

**Comment by mister-walter** [comment]
I've confirmed that 0.7.2 builds and installs successfully on my machine! Thanks so much!
